pkg/models/db.go

```diff
@@ -12,6 +12,29 @@ import (
 
 var log = &common.Log
 
+type DBModel interface {
```

Comment:
what's DBModel for?

Reply:
Oops... leftover from a different approach.